test/util/testfactory/blob.go

```diff
@@ -25,12 +25,12 @@ func GenerateRandomlySizedBlobs(count, maxBlobSize int) []types.Blob {
 		blobs = nil
 	}
 
-	sort.Sort(types.BlobsByNamespace(blobs))
+	blobs = SortBlobs(blobs)
 	return blobs
 }
 
 // GenerateBlobsWithNamespace generates blobs with namespace ns.
-func GenerateBlobsWithNamespace(count int, blobSize int, ns appns.Namespace) types.BlobsByNamespace {
+func GenerateBlobsWithNamespace(count int, blobSize int, ns appns.Namespace) []types.Blob {
```

Comment:
[non-blocking]
why not keep using the `types.BlobsByNamespace` which already implements the `sort` interface

Reply:
it was removed from core due to core not really being namespace aware, so yeah perhaps we should use that instead. atm I just tossed in a function